### ISSUE TITLE
tag new version and publish to Bower

### ISSUE THREAD

**Issue by srigi** [anacronw/js-fixtures#7]
Last tag is over one year old and there are still Jasmine deps. Please tag new version, so people can load latest code via Bower. Also publishing to Bower is good idea - people can find project via [bower-components](http://sindresorhus.com/bower-components/).

**Comment by anacronw** [comment]
I've used bower before - not sure why I didn't think of this, thanks for the reminder.

The tags that were in the repo came from jasmine-jquery completely.  Their versions no longer match mine of course.  Rather than starting a new repo, I decided to just wipe the old tags.  Good or bad, I hope I'll ever have to do that once here.  Starting from now I'll make sure to publish the tags as well.

Unfortunately, I typo'd the git endpoint upon registration and am requesting the change on their page.  I'll update when this is fixed.
